server/comm/transport/frame_coding.py

```diff
@@ -13,7 +13,7 @@ def write_byte(self, byte: int):
             self._write_slice(False)
 
     def write_slice_size(self, size: int):
-        assert size <= 0x3FFF
+        assert size <= 0x3FFFF
```

Comment:
Oh there is mistake here, it's suppose to be
```suggestion
        assert size <= 0x7FFF
```
16 383 -> 32 767 bytes

Reply:
changed

Reply:
Did you tested this though?

Reply:
Yes, no issues